### PR TITLE
Add ppc64le support

### PR DESCRIPTION
**Pull request by cs-zhang**

### CODE REVIEW

**Review by yussufsh — CHANGES_REQUESTED** [review]
Changes look good. However can we have the plugins downloaded to current directly instead of making changes to user files? Please refer to the Readme at https://github.com/ocp-power-automation/terraform-providers-power

Also need to rebase your commit by taking latest changes from devel.

**Comment by cs-zhang** [comment]
Where can I check what cause the check fail?

**Review by yussufsh — COMMENTED** [review]
These set of changes looks good. I will give it a try soon.
Could you please squash all the commits in 1?

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Here is [quick demo](https://asciinema.org/a/380928).
 ## Features
 
 * Simple script based installer to deploy OpenShift (4.5 onwards) cluster on PowerVS leveraging Infrastructure as Code (IaC) pattern
-* Supports multiple (64-bit) x86 platforms including Linux, Windows & Mac OSX
+* Supports multiple (64-bit) x86 platforms including Linux, Windows & Mac OSX, also Linux for ppc64le
```
Comment (bpradipt) [inline README.md:35]: Just change it to "Supports multiple (64-bit) platforms including Linux(x86_64/ppc64le), Windows & Mac OSX"

File: openshift-install-powervs [hunk 2]
```diff
@@ -13,7 +13,7 @@ limitations under the License.
 '
 #-------------------------------------------------------------------------
 set -e
-VERSION="v1.0.3"
+VERSION="v1.0.4"
```
Comment (yussufsh) [inline openshift-install-powervs:16]: Not intended?

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: openshift-install-powervs (hunk 2 above)
Comment (cs-zhang) [reply openshift-install-powervs:16]: I did not make this change, It comes from the merged from upstream.

**Comment by bpradipt** [comment]
@cs-zhang the code looks good to me.. I don't know why the commit shows this " yussufsh authored and cs-zhang committed 4 days ago"
May be something wrong when you rebased with the latest changes ?
@yussufsh any ideas ?

File: openshift-install-powervs (hunk 2 above)
Comment (yussufsh) [reply openshift-install-powervs:16]: May be it was added while resolving conflicts during merge/rebase.

Comment (cs-zhang) [reply openshift-install-powervs:16]: I have checked the `VERSION` on devel branch at github, it is 1.0.4. If it is OK, I'll push a new commit with signed-off-by.

Comment (cs-zhang) [reply openshift-install-powervs]: I have synced my devel branch with upstream devel branch, but the version still show changed. Could be the code in github need to be synced before I push my code? How to sync my fork to origin?

**Review by cs-zhang — COMMENTED** [review]
I should not squash all commits to one, that caused the version problem.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by yussufsh** [comment]
> 
> 
> I should not squash all commits to one, that caused the version problem.

The problem was that did not have the latest code at the time of commit. Anyways the code now look good and will on Power platform.

Thanks for your contribution.

**Review by yussufsh — APPROVED** [review]
lgtm

**Pull request merged**